src/main/java/org/apache/maven/archiver/MavenArchiver.java

```diff
@@ -703,4 +703,13 @@ private Artifact findArtifactWithFile( Set<Artifact> artifacts, File file )
         }
         return null;
     }
+
+    private String getArchiverVersion()
```

Comment:
Nit: static?

Reply:
I thought about this too, but didn't see a real benefit in unless it is public. Does it make sense to you to make it static?

Reply:
Yes. I think it is cleaner. That method does not depend on *this*.

I think spotbugs/sonarquce would have marked it as a bug :)